### PR TITLE
separated sales and refund datasets in weekly_revenues api

### PR DESCRIPTION
**Pull request by shreyas-satish**

### CODE REVIEW

File: boxoffice/models/payment.py [hunk 1]
```diff
@@ -79,3 +81,30 @@ class CURRENCY(LabeledEnum):
...
+def calculate_weekly_refunds(item_collection_ids, user_tz, year):
+    """
+    Calculates refunds per week for a given set of item_collection_ids in a given year,
+    in the user's timezone.
+    """
+    ordered_week_refunds = OrderedDict()
+    for year_week in Week.weeks_of_year(year):
+        ordered_week_refunds[year_week.week] = 0
+    start_at = isoweek_datetime(year, 1)
+    end_at = isoweek_datetime(year + 1, 1)
```
Comment (jace) [inline boxoffice/models/payment.py]: These two need the `user_tz` parameter.

File: boxoffice/models/line_item.py [hunk 2]
```diff
@@ -268,10 +268,10 @@ def calculate_weekly_sales(item_collection_ids, user_tz, year):
     week_sales = db.session.query('sales_week', 'sum').from_statement(db.text('''SELECT EXTRACT(WEEK FROM ordered_at)
         AS sales_week, SUM(final_amount) AS sum
         FROM line_item INNER JOIN item on line_item.item_id = item.id
-        WHERE status=:status AND item_collection_id IN :item_collection_ids
+        WHERE status IN :statuses AND item_collection_id IN :item_collection_ids
         AND ordered_at >= :start_at AND ordered_at < :end_at
```
Comment (jace) [inline boxoffice/models/line_item.py]: `user_tz` is not being used here in `start_at` and `end_at`.

File: boxoffice/models/payment.py [hunk 3]
```diff
@@ -79,3 +81,30 @@ class CURRENCY(LabeledEnum):
 
 class CURRENCY_SYMBOL(LabeledEnum):
     INR = (u"INR", u"₹")
```
Comment (jace) [inline boxoffice/models/payment.py]: These should be in single quotes as they are not translatable strings.

File: boxoffice/views/admin.py [hunk 4]
```diff
@@ -40,24 +41,31 @@ def org(organization):
...
+    if request.args.get('timezone') not in pytz.common_timezones:
+        return api_error(message='Unknown timezone. timezone is case-sensitive.', status_code=400)
 
     item_collection_ids = [item_collection.id for item_collection in organization.item_collections]
     year = int(request.args.get('year'))
-    weekly_sales = calculate_weekly_sales(item_collection_ids, user_timezone, year).items()
-    return api_success(result=weekly_sales, doc="Revenue per week for {year}".format(year=year), status_code=200)
+    user_timezone = request.args.get('timezone')
+
+    if request.args.get('refund') == 't':
```
Comment (jace) [inline boxoffice/views/admin.py]: Use `coaster.views.getbool` for this. Don't check for a `t` value.

File: boxoffice/models/payment.py (hunk 3 above)
Comment (shreyas-satish) [inline boxoffice/models/payment.py]: Fixed.

File: boxoffice/models/payment.py (hunk 1 above)
Comment (shreyas-satish) [inline boxoffice/models/payment.py]: Fixed.

File: boxoffice/models/line_item.py (hunk 2 above)
Comment (shreyas-satish) [inline boxoffice/models/line_item.py]: Fixed.

File: boxoffice/views/admin.py (hunk 4 above)
Comment (shreyas-satish) [inline boxoffice/views/admin.py]: Fixed.

**Pull request merged**